exchanges/poloniex/poloniex_futures.go

```diff
@@ -178,18 +193,37 @@ func (e *Exchange) CloseAtMarketPrice(ctx context.Context, symbol, marginMode, p
 		ClOrdID:      clientOrderID,
 		PositionSide: positionSide,
 	}
-	var resp *FuturesV3OrderIDResponse
-	return resp, e.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, request.UnAuth, http.MethodPost, "/v3/trade/position", nil, arg, &resp, true)
+	var resp *FuturesOrderIDResponse
+	err := e.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, request.UnAuth, http.MethodPost, "/v3/trade/position", nil, arg, &resp, true)
+	if err != nil {
+		return nil, err
+	}
+	if resp.Code != 0 {
+		return nil, fmt.Errorf("%w: code: %d message: %s", common.ErrNoResponse, resp.Code, resp.Message)
+	}
+	return resp, nil
 }
 
 // CloseAllAtMarketPrice close all orders at market price.
-func (e *Exchange) CloseAllAtMarketPrice(ctx context.Context) ([]FuturesV3OrderIDResponse, error) {
-	var resp []FuturesV3OrderIDResponse
-	return resp, e.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, request.UnAuth, http.MethodPost, "/v3/trade/positionAll", nil, nil, &resp, true)
+func (e *Exchange) CloseAllAtMarketPrice(ctx context.Context) ([]FuturesOrderIDResponse, error) {
+	var resp []FuturesOrderIDResponse
+	err := e.SendAuthenticatedHTTPRequest(ctx, exchange.RestSpot, request.UnAuth, http.MethodPost, "/v3/trade/positionAll", nil, nil, &resp, true)
+	if err != nil {
+		return nil, err
+	}
+	if len(resp) == 0 {
+		return nil, common.ErrNoResults
+	}
+	for _, r := range resp {
+		if r.Code != 0 {
+			return nil, fmt.Errorf("%w: code: %d message: %s", common.ErrNoResponse, r.Code, r.Message)
+		}
+	}
```

Comment:
We can just return the type if we are using CancelAllOrder func then we need to know what order failed, this can happen for all multiple order ops.